docs/source/network/network.md

```diff
@@ -297,10 +297,10 @@ to access. We can think of L1 as being **physically hosted** on P1, but
 add more peers to the channel.
 
 A key part of a P1's configuration is an X.509 identity issued by CA1 which
-associates P1 with organization R1. Once P1 is started, it can **join** channel
-C1 using the orderer O4. When O4 receives this join request, it uses the channel
-configuration CC1 to determine P1's permissions on this channel. For example,
-CC1 determines whether P1 can read and/or write information to the ledger L1.
+associates P1 with organization R1. The orderer O4 uses the channel configuration
```

Comment:
```suggestion
associates P1 with organization R1. When an organization administrator joins peer P1 to channel C1, and the peer starts pulling blocks from the orderer O4, the orderer uses the channel configuration
```

Reply:
Thanks. I felt I was dropping some bit of information.
Looks much better now